src/components/vedtak-liste/vedtak-liste.tsx

```diff
@@ -52,6 +50,13 @@ const VedtakListe = () => {
                     tomListeTekst={tekst('vedtak-liste.ingen-tidligere-soknader')}
                 />
 
+                <Lenke className="arkiverte_vedtak" href="https://www.nav.no/mine-saker/tema/SYK">
```

Comment:
Synes vi bør fikse det slik at det blir riktig i dev.

Reply:
Antar at du mener riktig url? Jeg lurer også på om vi skal ha chevron?

Reply:
Ja, egen url i dev. 

Det er uten chevron i sykepengesoknad https://sykepengesoknad.labs.nais.io/syk/sykepengesoknad/

Der finner du også url for dev